clients/vault/tests/vault_integration_tests.rs

```diff
@@ -772,7 +785,7 @@ async fn test_issue_overpayment_succeeds() {
 		join(
 			assert_event::<EndowedEvent, _>(TIMEOUT, user_provider.clone(), |x| {
 				if &x.who == user_provider.get_account_id() {
-					assert_eq!(x.amount, issue.amount * over_payment_factor);
+					// assert_eq!(x.amount, issue.amount * over_payment_factor);
```

Comment:
oh, we don't need this anymore?

Reply:
Wow, thanks a lot for noticing this. I actually commented this out during development for debugging purposes but forgot about it again. I added another [commit](https://github.com/pendulum-chain/spacewalk/pull/310/commits/dc8faa7716ee36398f34cd3badf8cb32c101840e) that solves a problem that is only apparent if the line is not commented out.